### ISSUE TITLE
Не всегда заполняет полностью задания с перетаскиванием нужного ответа

### ISSUE THREAD

**Issue by Destrolaric** [FutureXpo/Chrome-Extension-LMS-Answers#3]
![image](https://user-images.githubusercontent.com/20579256/83243172-368bdc00-a1a6-11ea-9734-f2aec1bf5fb6.png)

**Comment by FutureXpo** [comment]
Да, там много косяков, но изза того что я больше не занимаюсь в лмс, то и доделывать лень
Можешь сам в коде подправить, там несложно разобраться что к чему
